lxd/storage/drivers/driver_btrfs.go

```diff
@@ -491,7 +491,7 @@ func (d *btrfs) MigrationTypes(contentType ContentType, refresh bool, copySnapsh
 	if d.state.OS.RunningInUserNS {
 		var transportType migration.MigrationFSType
 
-		if contentType == ContentTypeBlock {
+		if contentType == ContentTypeBlock || contentType == ContentTypeISO {
```

Comment:
I'm seeing this pattern a lot in this PR:

```
if contentType == ContentTypeBlock || contentType == ContentTypeISO
```

Could we add a function somewhere called, perhaps, `isContentBlock() bool`? (or a better name :))